.circleci/config.yml

```diff
@@ -30,3 +30,16 @@ workflows:
               ignore: /.*/
             tags:
               only: /^v.*/
+
+      - architect/push-to-app-collection:
+          context: architect
+          name: push-to-vsphere-addons-app-collection
+          app_name: "cluster-api-cleaner-vsphere"
```

Comment:
I think this should be the ipam provider app

Reply:
fixed 🤦